servers/rendering_server.cpp

```diff
@@ -319,6 +319,72 @@ RID RenderingServer::get_white_texture() {
 	return white_texture;
 }
 
+float sign_zero(float m_v) {
```

Comment:
Not sure if this should live here; the SIGN implementation changed from 3.x to 4.0 where now SIGN returns 0 when 0 is passed in (which matches what glsl does but is not what we want here haha)